tests/test_AggregateNetworkData.cpp

```diff
@@ -0,0 +1,317 @@
+    BOOST_CHECK_EQUAL(static_cast<int>(networkData.getRNode().size()), ih[VI::NRNODE] * ih[VI::NODMAX]);
+    BOOST_CHECK_EQUAL(static_cast<int>(networkData.getRBran().size()), ih[VI::NRBRAN] * ih[VI::NODMAX]);
+
+    //INode-parameters
+    const auto& iNode = networkData.getINode();
+    auto start = 0*ih[VI::NINODE];
+    BOOST_CHECK_EQUAL(iNode[start + 0], 1);
+    BOOST_CHECK_EQUAL(iNode[start + 1], 1);
+    BOOST_CHECK_EQUAL(iNode[start + 2], 4);
+    BOOST_CHECK_EQUAL(iNode[start + 3], 0);
+    BOOST_CHECK_EQUAL(iNode[start + 4], 1);
+
+    start = 1*ih[VI::NINODE];
+    BOOST_CHECK_EQUAL(iNode[start + 0], 2);
+    BOOST_CHECK_EQUAL(iNode[start + 1], 2);
+    BOOST_CHECK_EQUAL(iNode[start + 2], 3);
+    BOOST_CHECK_EQUAL(iNode[start + 3], 0);
+    BOOST_CHECK_EQUAL(iNode[start + 4], 1);
+
+    start = 2*ih[VI::NINODE];
+    BOOST_CHECK_EQUAL(iNode[start + 0], 3);
+    BOOST_CHECK_EQUAL(iNode[start + 1], 4);
+    BOOST_CHECK_EQUAL(iNode[start + 2], 1);
+    BOOST_CHECK_EQUAL(iNode[start + 3], 0);
+    BOOST_CHECK_EQUAL(iNode[start + 4], 1);
+
+    start = 3*ih[VI::NINODE];
+    BOOST_CHECK_EQUAL(iNode[start + 0], 1);
+    BOOST_CHECK_EQUAL(iNode[start + 1], 7);
+    BOOST_CHECK_EQUAL(iNode[start + 2], 2);
+    BOOST_CHECK_EQUAL(iNode[start + 3], 1);
+    BOOST_CHECK_EQUAL(iNode[start + 4], 1);
+
+    start = 4*ih[VI::NINODE];
+    BOOST_CHECK_EQUAL(iNode[start + 0], 1);
+    BOOST_CHECK_EQUAL(iNode[start + 1], 8);
+    BOOST_CHECK_EQUAL(iNode[start + 2], 6);
+    BOOST_CHECK_EQUAL(iNode[start + 3], 0);
+    BOOST_CHECK_EQUAL(iNode[start + 4], 1);
+
+    start = 5*ih[VI::NINODE];
+    BOOST_CHECK_EQUAL(iNode[start + 0], 2);
+    BOOST_CHECK_EQUAL(iNode[start + 1], 9);
+    BOOST_CHECK_EQUAL(iNode[start + 2], 5);
+    BOOST_CHECK_EQUAL(iNode[start + 3], 0);
+    BOOST_CHECK_EQUAL(iNode[start + 4], 1);
+
+    //IBran-parameters
+    const auto& iBran = networkData.getIBran();
+    start = 0*ih[VI::NIBRAN];
+    BOOST_CHECK_EQUAL(iBran[start + 0], 1);
+    BOOST_CHECK_EQUAL(iBran[start + 1], 2);
+    BOOST_CHECK_EQUAL(iBran[start + 2], 3);
+
+    start = 1*ih[VI::NIBRAN];
+    BOOST_CHECK_EQUAL(iBran[start + 0], 2);
+    BOOST_CHECK_EQUAL(iBran[start + 1], 3);
+    BOOST_CHECK_EQUAL(iBran[start + 2], 8);
+
+    start = 2*ih[VI::NIBRAN];
+    BOOST_CHECK_EQUAL(iBran[start + 0], 3);
+    BOOST_CHECK_EQUAL(iBran[start + 1], 4);
+    BOOST_CHECK_EQUAL(iBran[start + 2], 7);
+
+    start = 3*ih[VI::NIBRAN];
+    BOOST_CHECK_EQUAL(iBran[start + 0], 5);
+    BOOST_CHECK_EQUAL(iBran[start + 1], 6);
+    BOOST_CHECK_EQUAL(iBran[start + 2], 3);
+
+    start = 4*ih[VI::NIBRAN];
+    BOOST_CHECK_EQUAL(iBran[start + 0], 6);
+    BOOST_CHECK_EQUAL(iBran[start + 1], 3);
+    BOOST_CHECK_EQUAL(iBran[start + 2], 8);
+
+    //ZNode-parameters
+    const auto& zNode = networkData.getZNode();
+    start = 0*ih[VI::NZNODE];
+    BOOST_CHECK_EQUAL(zNode[start + 0].c_str(), "B1      ");
```

Comment:
Try to reduce the dependency on counting exactly eight spaces; something like:

```C++
std::string pad8(const std::string& s) {
     return s + std::string( 8 - s.size(), ' ');   
}

const std::blank8 = "        ";
.....
BOOST_CHECK_EQUAL(zNode[start + 0], pad8("N1"));
BOOST_CHECK_EQUAL(zNode[start + 1], blank8);
```

Reply:
Fine - OK!